src/utils/utils_pipeline.h

```diff
@@ -112,6 +117,14 @@ struct ocf_pipeline_step {
 		.type = ocf_pipeline_step_terminator, \
 	}
 
+#define OCF_PL_STEP_CONDITIONAL(_hndl, _arg, _pred) \
+	{ \
+		.type = ocf_pipeline_step_conditional, \
+		.hndl = _hndl, \
+		.arg = _arg, \
```

Comment:
It's a bit more complicated. See `OCF_PL_STEP_ARG_INT`/`OCF_PL_STEP_ARG_PTR` for reference.